docs/Glossary/index.md

```diff
@@ -0,0 +1,251 @@
+
+### Peer
+
+A general term that encompasses both [Seeders](#seeder) and [Leechers](#leecher).
+
+### Perma-Seed
+
+Seeding your torrents permanently ("forever"). Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.
+
+### [Prowlarr](../Prowlarr/index.md)
+
+An application designed to help with automation and the management of your [trackers](#tracker) and [indexers](#indexer). One of a group of applications, including Lidarr, Prowlarr, and Sonarr, collectively known as the "arrs".
+
+---
+
+## Q
+
+### qBit Manage
+
+[qBit Manage](https://github.com/StuffAnThings/qbit_manage){:target="\_blank" rel="noopener noreferrer"} (also known as `qbm` or `qbitmanage` / `qb_manage`) is a program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes such as time and tracker.
+
+### qBittorrent
+
+!!!info
+
+    Also known as: qb, qbit, qbittorrent
+
+[qBittorrent](https://www.qbittorrent.org/){:target="\_blank" rel="noopener noreferrer"} is a free, open-source, cross-platform BitTorrent client. Please note that it is highly recommended to avoid new major and minor (X and X.X) versions until properly validated by the community, as these often contain significant changes and/or bugs.
+
+### Quality
+
+Refers to the specific release quality that describes both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.
+
+### Quality Definitions
+
+Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.
+
+### Quality Profile
+
+!!!info
+
+    Also known as: QP
+
+A defined group and preferential ordering of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.
+
+---
+
+## R
+
+### [Radarr](../Radarr/index.md)
+
+An application designed to help with automation and the management of your movie library. One of a group of applications, including Lidarr, Prowlarr, and Sonarr, collectively known as the "arrs".
+
+---
+
+## S
+
+### Seeder
+
+BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.
+
+### [Sonarr](../Sonarr/index.md)
+
+An application designed to help with automation and the management of your TV Show/Series library. One of a group of applications, including Lidarr, Prowlarr, and Radarr, collectively known as the "arrs".
+
+### Swarm
+
+The collective group of all [Peers](#peer) of a given torrent (this includes both [Seeders](#seeder) and [Leechers](#leecher)) who are actively sharing any portion of the torrent.
```

Comment:
```suggestion
The group of [Peers](#peer), collectively, on a specific torrent (this includes both [Seeders](#seeder) and [Leechers](#leecher)) who are actively sharing any portion of the torrent.

This can also refer to the surge of users downloading a torrent shortly after its release.